vm/Cargo.toml

```diff
@@ -57,10 +56,10 @@ keccak = { workspace = true }
 hashbrown = { workspace = true }
 anyhow = { workspace = true }
 thiserror-no-std = { workspace = true }
+stark-felt = { git = "https://github.com/lambdaclass/types-rs.git", rev = "ea52b64a52fd832222adf400feb2dd3b4dc286a2", default-features = false, features = ["serde"] }
```

Comment:
I comment this just a as reminder, we should make a release in the type-rs crate

Reply:
We have the release we need I think, I just forgot to update! 🤦